dhis-2/dhis-services/dhis-service-dxf2/src/main/java/org/hisp/dhis/dxf2/metadata/MetadataExportParams.java

```diff
@@ -246,4 +253,19 @@ public boolean getSkipSharing()
     {
         return this.skipSharing;
     }
+
+    public boolean isExportWithDependencies()
+    {
+        return objectExportWithDependencies != null;
+    }
+
+    public IdentifiableObject getObjectExportWithDependencies()
+    {
+        return objectExportWithDependencies;
```

Comment:
I am a bit confused by the naming here. As far as I understand the object is the dependency. Wouldn't it be more clear to call this field simply "dependency" or "exportDependency"? (The `isExportWithDependencies` makes sense as is)